tests/models/test_model_names.py

```diff
@@ -40,10 +40,18 @@ def get_model_names(model_name_type: Any) -> Iterator[str]:
     openai_names = [f'openai:{n}' for n in get_model_names(OpenAIModelName)] + [
         n for n in get_model_names(OpenAIModelName) if n.startswith('o1') or n.startswith('gpt') or n.startswith('o3')
     ]
+    deepseek_names = ['deepseek:deepseek-chat', 'deepseek:deepseek-reasoner']
```

Comment:
We don't have a concept of DeepSeek models in the source code, only DeepSeek provider. Should we add it? But where?

Reply:
What do you mean? is it something other than `deepseek-chat` or `deepseek-reasoner`?

Reply:
No, only those 2. I mean that we have `OpenAIModelName`, and others, but we don't have for DeepSeek.